src/Components/Details.js

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from 'react';
```

Comment:
Mmm ok so now you use `React.Component`, can you please delete the unused `{ Component }` import?